README.md

```diff
@@ -163,13 +206,12 @@ git push
 
 After you push, the FoundriesFactory will build a new target. Once built, any registered device will update over the air.
 
-
 ### SSH
 
 Once connected to the network you can log in over ssh if desired:
 
 ```bash
-ssh fio@qcs6490-rb3gen2-vision-kit.local
+ssh fio@qcm6490.local
```

Comment:
qcs6490-rb3gen2-vision-kit.local is the correct name, it seems it wasn't rebased on the current naming.